styx-common/src/main/java/com/spotify/styx/api/cli/ActiveStatesPayload.java

```diff
@@ -52,20 +56,33 @@
     public abstract String state();
 
     @JsonProperty
+    public abstract StateData stateData();
```

Comment:
This addition will break older CLI artifacts, even though it's a backwards compatible change.

That is because the `ActiveState` type was missing the `@JsonIgnoreProperties(ignoreUnknown = true)` annotation which I added now.